### DISCUSSION TITLE
tokens are gone

### DISCUSSION THREAD

**Discussion by pribridd** [iotaledger/firefly#4717] [Help & Support]
until version 1.4.2 my smr token appeared, after updating to the next version onwards they disappeared and the smr field does not appear, only iota and asmb.
I need help to recover these tokens I don't know what happened

**Comment by michaelwants** [comment]
I had the same problem, but it was my Mi that disappeared

**Reply by Laurette777** [reply]
Same. Anyone know if it's fixed yet? My SMR missing.

**Comment by begonaalvarezd** [comment]
Quick information about ASMB and SMR:
- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version (V2.0.7 as of today) in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app and claim your rewards, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/
